Add Turnstile Apperance & its paramter

MisterJEFFREY opened the pull request:
Added Turnstile Appearance & its parameter to multiple assets in the BlazorTurnstile nuget package.

MisterJEFFREY committed:
Added Turnstile Apperance & its paramter to multiple asstes in the BlazorTurnstile nuget package.

MisterJEFFREY commented:
No extra comments for this PR yet.

MisterJEFFREY committed:
Update README.md

MichelJansson commented on src/BlazorTurnstile/TurnstileAppearance.cs:
```diff
@@ -0,0 +1,12 @@
+public enum TurnstileAppearance {
+    always,
+    execute,
+    [EnumMember(Value = "interaction-only")]
```
It seems like the `EnumMemeberAttribute` isn't yet supported and the sample won't work without additional converters. The interaction only value is not serialized correctly which makes turnstile js to throw an exception.  Did you get this working on your end?

MisterJEFFREY replied:
Thanks for looking into this, @MichelJansson.
I apologize for this hiccup and you are correct. The serialization does not respect enumerations without converters and went with a more classical approach:

```javascript
    // Convert non-callback parameter(s) with values containing "_" to expected values with "-" because serializations do not respect enumerations.
    for (let key in parameters) {
         if (!["call-back", "action", "cData", "response-field-name"].includes(key)) {
            if (parameters[key] && typeof parameters[key] === 'string' && parameters[key].includes("_")) {
                parameters[key] = parameters[key].replace(/_/g, "-");
            }
        }
    }
``` 

This should work for any parameters with dashes in non-custom values.

Please let me know if this works for you.

MichelJansson replied:
Thanks for the suggestion, but I'd like to stay in .NET land as far as possible, so I made a hole "re-take" on the serialization that now is kebab-case by default for the entire parameter object tree. This removes the need for attributes and js.

MichelJansson reviewed (commented):
Thanks for this!
The "Interaction only" appearance does not work on my end though (see separate comment).

MisterJEFFREY committed:
-Corrected iincorrect serialization for the Appearance attribute containing value with "-" inside Interop.js
-Set Apperance attribute to always inside Index.razor
-Updated readme.md w/ new Appearance attribute

MisterJEFFREY committed:
Resolving merge conflict

MisterJEFFREY committed:
-Added the "response-field-name" paramater to be excluded from the potential "_" check inside interop.js because its value can be customer defined.

MichelJansson commented:
I went ahead and cherry-picked your initial changes (adding you as a co-author) to save you the hassle of sorting out multiple merges, so I'm closing this PR - hope you don't mind!

Thanks for the help!
